arlo/messages.py

```diff
@@ -427,6 +454,18 @@ def from_json(json_data):
                 "targetbps":1024000,
                 "cbrbps":1024000
                 },
+            # I'm not sure what subscription quality is
```

Comment:
Subscription quality were some slightly higher settings when I enabled the subscription for Arlo Smart. It didn't seem to make much difference in reality.